helpers/lib_tasks_docker_release.py

```diff
@@ -62,6 +62,207 @@ def _get_dev_version(version: str, container_dir_name: str) -> str:
     return dev_version
 
 
+def _create_multiarch_builder(
+    ctx: Any,
+    opts: str,
+    multi_arch: str,
+    build_args: str,
+    image_local: str,
+    dockerfile: str,
+) -> None:
+    """
+    Create a multi-arch builder and build the image for multiple architectures.
+
+    :param ctx: invoke context
+    :param opts: build options (e.g., --no-cache)
+    :param multi_arch: target architectures to build for
+    :param build_args: build arguments for the Docker build command
+    :param image_local: name of the local image to build
+    :param dockerfile: path to the Dockerfile to use for building
+    """
+    # Login to AWS ECR because for multi-arch we need to build the local
+    # image remotely.
+    hlitadoc.docker_login(ctx)
+    # Create a multi-arch builder.
+    platform_builder = "multiarch_builder"
+    cmd = rf"""
+    docker buildx rm {platform_builder}
+    """
+    # We do not abort on error since the platform builder might be present
+    # or not from previous executions.
+    hsystem.system(cmd, abort_on_error=False)
+    cmd = rf"""
+    docker buildx create \
+        --name {platform_builder} \
+        --driver docker-container \
+        --bootstrap \
+        && \
+        docker buildx use {platform_builder}
+    """
+    hlitauti.run(ctx, cmd)
+    # Build.
+    # Compress the current directory (in order to dereference symbolic
+    # links) into a tar stream and pipes it to the `docker build` command.
+    # See HelpersTask197.
+    cmd = rf"""
+    tar -czh . | DOCKER_BUILDKIT={DOCKER_BUILDKIT} \
+        time \
+        docker buildx build \
+        {opts} \
+        --push \
+        --platform {multi_arch} \
+        {build_args} \
+        --tag {image_local} \
+        --file {dockerfile} \
+        -
+    """
+    hlitauti.run(ctx, cmd)
+    # Pull the image from registry.
+    cmd = rf"""
+    docker pull {image_local}
+    """
+    hlitauti.run(ctx, cmd)
+
+
+def _list_image(ctx: Any, repository_tag: str) -> None:
+    """
+    List a specific Docker image with the given tag.
```

Comment:
Let's not duplicate parameters in the description. 
`List Docker image` would be enough.